conjure-java-core/src/integrationInput/java/com/palantir/product/SafeExternalLongExample.java

```diff
@@ -158,68 +165,68 @@ public Builder from(SafeExternalLongExample other) {
+        public Builder safeExternalLongValue(@Safe long safeExternalLongValue) {
             checkNotBuilt();
             this.safeExternalLongValue = safeExternalLongValue;
             this._safeExternalLongValueInitialized = true;
             return this;
         }
 
         @JsonSetter(value = "optionalSafeExternalLong", nulls = Nulls.SKIP)
-        public Builder optionalSafeExternalLong(@Nonnull Optional<? extends Long> optionalSafeExternalLong) {
+        public Builder optionalSafeExternalLong(@Safe @Nonnull Optional<? extends Long> optionalSafeExternalLong) {
             checkNotBuilt();
             this.optionalSafeExternalLong = Preconditions.checkNotNull(
                             optionalSafeExternalLong, "optionalSafeExternalLong cannot be null")
                     .map(Function.identity());
             return this;
         }
 
-        public Builder optionalSafeExternalLong(long optionalSafeExternalLong) {
+        public Builder optionalSafeExternalLong(@Safe long optionalSafeExternalLong) {
             checkNotBuilt();
             this.optionalSafeExternalLong = Optional.of(
                     Preconditions.checkNotNull(optionalSafeExternalLong, "optionalSafeExternalLong cannot be null"));
             return this;
         }
 
         @JsonSetter(value = "safeExternalLongList", nulls = Nulls.SKIP, contentNulls = Nulls.FAIL)
-        public Builder safeExternalLongList(@Nonnull Iterable<? extends Long> safeExternalLongList) {
+        public Builder safeExternalLongList(@Safe @Nonnull Iterable<? extends Long> safeExternalLongList) {
             checkNotBuilt();
             this.safeExternalLongList = ConjureCollections.newArrayList(
                     Preconditions.checkNotNull(safeExternalLongList, "safeExternalLongList cannot be null"));
             return this;
         }
 
-        public Builder addAllSafeExternalLongList(@Nonnull Iterable<? extends Long> safeExternalLongList) {
+        public Builder addAllSafeExternalLongList(@Safe @Nonnull Iterable<? extends Long> safeExternalLongList) {
             checkNotBuilt();
             ConjureCollections.addAll(
                     this.safeExternalLongList,
                     Preconditions.checkNotNull(safeExternalLongList, "safeExternalLongList cannot be null"));
             return this;
         }
 
-        public Builder safeExternalLongList(long safeExternalLongList) {
+        public Builder safeExternalLongList(@Safe long safeExternalLongList) {
             checkNotBuilt();
             this.safeExternalLongList.add(safeExternalLongList);
             return this;
         }
 
         @JsonSetter(value = "safeExternalLongSet", nulls = Nulls.SKIP, contentNulls = Nulls.FAIL)
-        public Builder safeExternalLongSet(@Nonnull Iterable<? extends Long> safeExternalLongSet) {
+        public Builder safeExternalLongSet(@Safe @Nonnull Iterable<? extends Long> safeExternalLongSet) {
```

Comment:
ideally this would be `@Nonnull Iterable<@Safe ? extends Long>` but there is a [bug](https://github.com/square/javapoet/issues/959) in poet that makes that impossible. i think this is a pretty good workaround.

Reply:
👍 looks reasonable to me